ee-feature-pack/common/pom.xml

```diff
@@ -1009,6 +1009,17 @@
             </exclusions>
         </dependency>
 
+        <dependency>
```

Comment:
Did something fail without this? This module doesn't build anything so should not need transitive deps of anything it declares. Code that consumes this, e.g.https://github.com/wildfly/wildfly/blob/main/ee-feature-pack/galleon-feature-pack/pom.xml#L396 is getting this dependency info imported from elsewhere, e.g. https://github.com/wildfly/wildfly/blob/main/ee-feature-pack/galleon-feature-pack/pom.xml#L364